spec/rubocop/cop/style/access_modifier_declarations_spec.rb

```diff
@@ -704,6 +704,12 @@ class TestThree
         RUBY
       end
 
+      it "does not register an offense for single #{access_modifier}", skip: :FIXME do
```

Comment:
It would be preferable to resolve the `skip: :FIXME`.

Reply:
Sorry for not marking it as draft -- it's waiting for https://github.com/rubocop/rubocop/pull/14170